installation/release-notes-main/release-notes-9-0-1.adoc

```diff
@@ -0,0 +1,74 @@
+- Documented instructions for patch update from {VERSION} to {VERSION901}
+- Delivered Migration guide for Version 9.0
+- Delivered Security bulletin for {CANVAS} and runtime libraries
+- Support for Quarkus Version 2.16.12
+
+== System requirements
+
+This section highlights the supported version updates in Version {VERSION901} fix release. 
+You can access the detailed system requirements from the IBM Support site link for {PRODUCT} {VERSION901} fix release.
+See https://spcr-prod-preview.dal1a.cirrus.ibm.com/software/reports/compatibility/clarity-reports/report/html/softwareReqsForProduct?deliverableId=839878FFE5974B07B85E83D0B1D368B0&osPlatforms=Linux%7CWindows&duComponentIds=S012%7CS011%7CD014%7CC013&mandatoryCapIds=25&optionalCapIds=71%7C26%7C1
+
+
+== Fix lists
+The summary list of fixes that are included in Version {VERSION901} fix release.
+
+- git-cors-proxy-image and kie-sandbox-image on macOS ARM processors
+- MvelCompiler is not coercing type double apache or incubator-kie-issues
+- Usage of appropriate BigDecimal method to convert to primitive type apache or incubator-kie-issues
+- {CANVAS} image uses the root filesystem on the container to store its assets and configuration files
+- {PRODUCT_SHORT} {VERSION} runtime image fails to run on Kubernetes with readOnlyRootFilesystem: true due to swidtag creation
+
+=== Licensing Model update
+
+- ILMT Libraries does not work on native builds
+
+== Known limitations
+
+To review the most current known limitations in detail, see Known limitations in {PRODUCT_LONG} {VERSION901} fix releases. 
+The issues that arise will be reported through this source.
+
+== Download documents
+
+The document contains information for obtaining {PRODUCT_LONG} ({PRODUCT_SHORT}) {VERSION}, earlier release and its fix releases, currently Version {VERSION901}. 
+{PRODUCT_SHORT} versions are designated with a numerical Major.Minor.Patch following Open Source community Semantic Versioning principle. 
+Fixes for on-premise installs are available through IBM Fix Central, and {CANVAS} container images are available through Quay Container Registry.
+For more inforamation, see https://www.ibm.com/support/pages/node/7001673
+
+== Upgrade {PRODUCT_SHORT} from Version 9.0.0 to Version 9.0.1
+
+To upgrade {PRODUCT_SHORT} from Version 9.0.0 to Version 9.0.1, follow these steps:
+
+- Update QUARKUS Version to {QUARKUS_VERSION_SHORT_12_FINAL}
+- Update pom file from their projects with Version 9.0.1 {PRODUCT_SHORT} BOM
+- For direct library components, use Version 8.40.1 for drools or Version 1.40.2 for kogito libraries.
+- Rebuild projects and re-deploy as regular software rollout.
+
+For Canvas:
+
+The new containers are the drop-in replacements for the existing. 
+You can use https://docs.openshift.com/container-platform/4.14/applications/deployments/route-based-deployment-strategies.html#deployments-blue-green_route-based-deployment-strategies[blue/green deployment]
```

Comment:
"You can use blue/green deployment" missing the word in the end: "strategies."

Reply:
Fixed